src/Details/Details.js

```diff
@@ -69,7 +70,7 @@ const Details = () => {
             )
           })}
       </div>
-      : <p>loading ...</p>}
+      : <Error error={error}/>}
```

Comment:
I'm seeing this render briefly on initial load of main page and details view - you may also want a state of `isLoading` that will fallback to a loading view until the data is ready to be displayed.